src/modules/Kdump.rb

```diff
@@ -1214,11 +1190,7 @@ def remove_offsets!(values)
     def write_fadump_boot_param
       if system.supports_fadump?
         # If fdump is selected and we want to enable kdump
-        if using_fadump? && @add_crashkernel_param
-            value = "on"
-        else
-            value = nil
-        end
+        value = ("on" if using_fadump? && @add_crashkernel_param)
```

Comment:
Just a question: are the parenthesis really needed? Why?

Reply:
no, this should not be needed.